### PR TITLE
Drink menu added

### PR DESCRIPTION
**Pull request by Ashu-4u**
- Added Drinks to the Vikas Food Plaza

### CODE REVIEW

File: src/main/java/org/tektutor/menuwebservice/MenuController.java [hunk 1]
```diff
@@ -34,6 +29,20 @@ public String sayHello(@RequestParam String firstName, @RequestParam String last
 		starterList.put(105, "Paneer Tikka");
 		return starterList;
 	}
+	
+
+	@GetMapping("/Drinks")
```
Comment (pracagra) [inline src/main/java/org/tektutor/menuwebservice/MenuController.java]: @Ashu-4u Can we have a good naming like `getDrinks` ? 

Please share your thoughts on this 
@regarmukesh3g @shradha1404

File: src/main/java/org/tektutor/menuwebservice/MenuController.java [hunk 2]
```diff
@@ -16,13 +16,8 @@
 	public String sayWelcome(@RequestParam String firstName,@RequestParam String secondName) {
 		return "Welcome to Vikas Maggi Plaza, " + firstName + " " + secondName;
 	}
-	
-	@GetMapping("/message")
-	@ResponseBody
-	public String sayHello(@RequestParam String firstName, @RequestParam String lastName) {
```
Comment (regarmukesh3g) [inline src/main/java/org/tektutor/menuwebservice/MenuController.java]: Not necessary

File: src/main/java/org/tektutor/menuwebservice/MenuController.java (hunk 1 above)
Comment (Ashu-4u) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: I'll change the comments ASAP, and will inform soon.

File: src/main/java/org/tektutor/menuwebservice/MenuController.java (hunk 2 above)
Comment (yogeshvar) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: @regarmukesh3g  what's not necessary ? Could you please elaborate.? 

cc - @Ashu-4u

Comment (regarmukesh3g) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: There is already a method for greeting customers, another method is not needed.

File: src/main/java/org/tektutor/menuwebservice/MenuController.java (hunk 1 above)
Comment (pracagra) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: @Ashu-4u do it as soon as possible

Comment (pracagra) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: @Ashu-4u  dont play games.concentrate on your work.

File: src/main/java/org/tektutor/menuwebservice/MenuController.java (hunk 2 above)
Comment (Ashu-4u) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: @regarmukesh3g I have not removed any methods. I think it is done by somebody else. Please confirm.

File: src/main/java/org/tektutor/menuwebservice/MenuController.java (hunk 1 above)
Comment (regarmukesh3g) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: @pracagra  Yes, It's our project. We can name whatever we want.

Comment (Ashu-4u) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: @pracagra I have resolves the issue please check it and reply if any further improvements are required.

Comment (pracagra) [reply src/main/java/org/tektutor/menuwebservice/MenuController.java]: @Ashu-4u   change yourself

**Pull request merged**